Components/CommentsList.jsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getCommentsByArticleId } from "../api";
+import { SingleCommentCard } from "./SingleCommentCard";
+import loadingGif from '../src/assets/loading.gif'
+
+export const CommentsList = () => {
+    const { article_id } = useParams();
+    const [comments, setComments] = useState([]);
+    const [isLoading, setIsLoading] = useState(true);
+
+    useEffect(() => {
+        getCommentsByArticleId(article_id)
+        .then((comments) => {
+            setComments(comments)
+            setIsLoading(false)
+        })
+        .catch((err) => {
+            console.log(err)
```

Comment:
It would be great if you had some error state rather than just console logging it!